Scripts/Modules/SnappingModule/SnappingModule.cs

```diff
@@ -725,40 +725,40 @@ void SetupUI()
 			handle.hoverEnded += (baseHandle, data) => { text.text = m_Preferences.disableAll ? "Snapping disabled" : "Snapping enabled"; };
 
 			var groundSnappingUI = m_SnappingModuleSettingsUI.groundSnappingEnabled;
-			groundSnappingUI.isOn = groundSnappingEnabled;
+			groundSnappingUI.isOn = m_Preferences.groundSnappingEnabled;
```

Comment:
I don't see how this fixes anything? The properties already routed to m_Preferences.